rosdep/base.yaml

```diff
@@ -5938,6 +5938,7 @@ libttspico:
   debian: [libttspico-dev, libttspico-data, libttspico-utils, libttspico0]
   ubuntu: [libttspico-dev, libttspico-data, libttspico-utils, libttspico0]
 libturbojpeg:
+  alpine: [libjpeg-turbo]
```

Comment:
Should be `-dev`?

Reply:
Yes, I just noticed that :bow: https://github.com/alpine-ros/rosdistro/pull/141